### ISSUE TITLE
Geometry optimization

### ISSUE THREAD

**Issue by SebWouters** [psi4/psi4public#66]
Hi,

The input file for the problem described below:

    molecule C6H6 {
     0 1
        C            0.0                    1.855     0.0
        C            0.0                   -1.855     0.0
        C            1.6064771240201337     0.9275    0.0
        C            1.6064771240201337    -0.9275    0.0
        C           -1.6064771240201337     0.9275    0.0
        C           -1.6064771240201337    -0.9275    0.0
        H            0.0                    3.289     0.0
        H            0.0                   -3.289     0.0
        H            2.848357553047019      1.6445    0.0
        H            2.848357553047019     -1.6445    0.0
        H           -2.848357553047019      1.6445    0.0
        H           -2.848357553047019     -1.6445    0.0
     units au
    }
    
    set basis          cc-pVDZ
    set reference      rhf
    set e_convergence  1e-12
    set d_convergence  1e-12
    set ints_tolerance 0.0
    
    optimize('b3lyp')

When you run the geometry optimization in the /tmp folder, the geometry of the SCF doesn't get updated:

First SCF:

    Geometry (in Bohr), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z       
    ------------   -----------------  -----------------  -----------------
           C         -0.000000000000     1.855000000000     0.000000000000
           C          0.000000000000    -1.855000000000     0.000000000000
           C          1.606477124020     0.927500000000     0.000000000000
           C          1.606477124020    -0.927500000000     0.000000000000
           C         -1.606477124020     0.927500000000     0.000000000000
           C         -1.606477124020    -0.927500000000     0.000000000000
           H         -0.000000000000     3.289000000000     0.000000000000
           H          0.000000000000    -3.289000000000     0.000000000000
           H          2.848357553047     1.644500000000     0.000000000000
           H          2.848357553047    -1.644500000000     0.000000000000
           H         -2.848357553047     1.644500000000     0.000000000000
           H         -2.848357553047    -1.644500000000     0.000000000000

At the end of optking:

        Writing optimization data to binary file.
        Structure for next step:
        Cartesian Geometry (in Angstrom)
            C    -0.0000000000   0.9816237969   0.0000000000
            C     0.0000000000  -0.9816237969   0.0000000000
            C     0.8501111451   0.4908118984   0.0000000000
            C     0.8501111451  -0.4908118984   0.0000000000
            C    -0.8501111451   0.4908118984   0.0000000000
            C    -0.8501111451  -0.4908118984   0.0000000000
            H    -0.0000000000   1.7404639720   0.0000000000
            H     0.0000000000  -1.7404639720  -0.0000000000
            H     1.5072860141   0.8702319860   0.0000000000
            H     1.5072860141  -0.8702319860   0.0000000000
            H    -1.5072860141   0.8702319860   0.0000000000
            H    -1.5072860141  -0.8702319860   0.0000000000

The new SCF:

    Geometry (in Bohr), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z       
    ------------   -----------------  -----------------  -----------------
           C         -0.000000000000     1.855000000000     0.000000000000
           C          0.000000000000    -1.855000000000     0.000000000000
           C          1.606477124020     0.927500000000     0.000000000000
           C          1.606477124020    -0.927500000000     0.000000000000
           C         -1.606477124020     0.927500000000     0.000000000000
           C         -1.606477124020    -0.927500000000     0.000000000000
           H         -0.000000000000     3.289000000000     0.000000000000
           H          0.000000000000    -3.289000000000     0.000000000000
           H          2.848357553047     1.644500000000     0.000000000000
           H          2.848357553047    -1.644500000000     0.000000000000
           H         -2.848357553047     1.644500000000     0.000000000000
           H         -2.848357553047    -1.644500000000     0.000000000000

In other folders (than /tmp) it works fine. Maybe change the naming scheme?

Best,
Sebastian

**Comment by CDSherrill** [comment]
Dear Sebastian,

Which version of Psi is this, beta5?  We have been having a little trouble
reproducing the problem.

Also, do you have a .psi4rc file, and if so, what's in it?

Thanks,
David

**Comment by SebWouters** [comment]
Dear David,

It's psi4 beta 5, and I don't have a .psi4rc file. I use Ubuntu 14.04.1 LTS, have compiled psi4 with the intel compiler, with MKL, with boost from the debian package repository, and with the plugin option on. For the rest I didn't set any options.

Best regards,
Sebastian

**Comment by jturney** [comment]
Hi Sebastian,

I'm unable to reproduce the error. Could you send me the output file?

**Comment by SebWouters** [comment]
Hi Justin,

My /tmp was erased, so it took me a while before being able to reproduce the error.
What was happening is the following:

- I started off with a really bad geometry and a certain order of C an H atoms.
- After a while, I killed psi4 to generate a new input file with D2h symmetry. Therefore I reordered the C and H atoms to corresponding D2h pairs.
- So in the folder (the problem would happen in any folder), filename.moleculename.intco was still there from the previous atom ordering.
- Psi4 tried to combine the new molecule with the old filename.moleculename.intco, and it got nowhere. It restrained the update steps to negligibly small displacements.

==> So how about changing filename.moleculename.intco > filename.moleculename.pid.intco ?

Best,
Sebastian
